single_tech_samples/storage/sample1_data_lifecycle/terraform/README.md

```diff
@@ -0,0 +1,190 @@
+# Terraform Script to create Storage Account , Containers and Data Lifecycle Rules at scale in Azure <!-- omit in toc -->
+
+## Contents <!-- omit in toc -->
+
+- [1. Solution Overview](#1-solution-overview)
+  - [1.1. Scope](#11-scope)
+  - [1.2. Use Case](#12-use-case)
+  - [1.3. Technologies used](#14-technologies-used)
+- [2. How to use this sample](#2-how-to-use-this-sample)
+  - [2.1. Prerequisites](#21-prerequisites)
+    - [2.1.1 Software Prerequisites](#211-software-prerequisites)
+  - [2.2. Setup and deployment](#22-setup-and-deployment)
+  - [2.3. Deployed Resources](#23-deployed-resources)
+  - [2.4. Deployment validation and Execution](#24-deployment-validation-and-execution)
+  - [2.5. Clean-up](#25-clean-up)
+- [3. Troubleshooting](#3-troubleshooting)
+
+## 1. Solution Overview
+
+With wide scale adaptation of Infrastructure As Code (IAC), many a times we would need to create storage accounts and corresponding entities dynamically using Infra As Code (IAC) Scripts.
```

Comment:
Suggest rewrite:

The purpose of this solution is to demonstrate the creation of Azure storage accounts/containers and configuring the data lifecycle rules at scale using Terraform based IaC templates.

The solution takes details about the storage accounts, containers and lifecycle rules as JSON configuration and apply it in a loop which makes it flexible, extensible and scalable to accommodate any number of storage accounts, containers and data life cycle rules.

Reply:
Done